kube-runtime/src/lib.rs

```diff
@@ -38,3 +38,34 @@ pub use watcher::{metadata_watcher, watcher};
 #[cfg(feature = "unstable-runtime-predicates")]
 pub use utils::{predicates, Predicate};
 pub use wait::conditions;
+
+pub mod prelude {
+    //! A "prelude" for kube runtime crate. Reduces the number of duplicated imports.
+    //!
+    //! This prelude is similar to the standard library's prelude in that you'll
+    //! almost always want to import its entire contents, but unlike the
+    //! standard library's prelude you'll have to do so manually:
+    //!
+    //! ```
+    //! use kube_runtime::prelude::*;
+    //! ```
+    //!
+    //! The prelude may grow over time as additional items see ubiquitous use.
+
+    pub use crate::reflector::Lookup as _;
+}
+
+#[cfg(feature = "unstable-runtime-predicates")]
+pub mod prelude_unstable {
```

Comment:
I don't think we need to separate stable and unstable preludes if we feature gate within the main prelude.

WatchStreamExt is generally not unstable. It has unstable methods (which are opt-in).

Reply:
Makes sense, it is just a new level of meta for traits and feature gates for me, and shadowing of the import at the looks of it does not seem to be happening. But if I understand correctly a lib re-exporting `WatchStreamExt` with `ws` feature enabled by default will show some methods under this feature, even if an end user crate does not seem to have it enabled. Also wasn’t able to find a better name. Also also I’m likely wrong, this needs testing. Will update it anyway soon 👀